int32 goalId
---
float32 bearing
bool is_valid
